packages/orbit-components/src/Seat/index.tsx

```diff
@@ -22,8 +22,8 @@ const Seat = ({
   id,
   price,
   label,
-  title = "Seat",
-  description = "Presents options for seating",
+  title,
+  description,
```

Comment:
Without the default values, wouldn't it make more sense to render `title` and `desc` conditionally?

Reply:
Yes